src/main/java/ua/danit/final_project/repositories/WorkShiftRepository.java

```diff
@@ -4,6 +4,10 @@
 import org.springframework.stereotype.Repository;
 import ua.danit.final_project.entities.WorkShift;
 
+import java.util.Date;
+import java.util.List;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.util.List'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=85&issues=AWWG_p8CX2sY87p04Y79&open=AWWG_p8CX2sY87p04Y79)

[ISSUE_KEY]: AWWG_p8CX2sY87p04Y79